### ISSUE TITLE
kfctl doesn't build with clean build because thrift repo is unavailable

### ISSUE THREAD

**Issue by jlewi** [kubeflow/kubeflow#4095]
/kind bug

**What steps did you take and what happened:**
See kubeflow/manifests#350 where kfctl build fails in a presubmit.

I was able to reproduce it locally

```
git checkout v0.6-branch
mkdir -p /tmp/newgo
export GOPATH=/tmp/newgo
cd bootstrap
make clean
make build-kfctl
```

Here we set the GOPATH to a clean directory to force all dependencies to be pulled.

The build fails with

```
go: git.apache.org/thrift.git@v0.0.0-20180902110319-2566ecd5d999: git fetch -f origin refs/heads/*:refs/heads/* refs/tags/*:refs/tags/* in /tmp/go/pkg/mod/cache/vcs/83dba939f95a790e497d565fc4418400145a1a514f955fa052f662d56e920c3e: exit status 128:
	fatal: unable to access 'https://git.apache.org/thrift.git/': Failed to connect to git.apache.org port 443: Connection timed out
```

It looks like lots of people are reporting similar issues; e.g.
hashicorp/terraform#22664

**Comment by jlewi** [comment]
Adding 

```
replace git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999
```
to go.mod appears to fix the issue.

**Comment by jlewi** [comment]
Looks like the fix is already on master.
https://github.com/kubeflow/kubeflow/blob/master/bootstrap/go.mod#L62

**Comment by jlewi** [comment]
Fixed by #4097
